composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
We made the concept of soft-require and now we should remove it.

The require should always specify the versions you now compatible. Here we now that 3.3 may be incompatible, therefore we have to require `~3.2.0`. That's also how it is done in Symfony components:
- In Symfony `2.5` the require was `~2.3` because `3.0` may break soemthing: https://github.com/symfony/symfony/blob/2.5/src/Symfony/Component/Config/composer.json#L20
- But in Symfony `2.8` we now that the component is fully compatible with Symfony `3.0` but we can't say anything for `3.1`: Symfony `3.1` has to be comaptible with Symfony `3.0` not necessarily with Symfony `2.8`, therefore the require is `~2.3 | ~3.0.0`: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Config/composer.json#L20
- And in Symfony `3.0` it is `~2.8 | ~3.0` because before `2.8` it's incompatible with `3.0` and it must be compatible with any upper version in the `3.x` branch

For phpBB you just have to replace `2.8` by `3.2` and `3.0` by `3.3`